### PR TITLE
Update README with multi account bootstrap instructions and tidy maintenance steps

### PR DESCRIPTION
**Pull request by worldofdub**
*Issue #, if available:*

*Description of changes:*

Updated instructions for maintenance steps to clarify where to run deployDev.sh

Updated bootstrapping guidance with the option for the prod AWS account trusting the pipeline/dev AWS account.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -267,6 +267,10 @@ This deployment option does create resources beyond what is created in the ‘St
 
 `cdk bootstrap aws://<aws-account-id>/<aws-region> --cloudformation-execution-policies arn:aws:iam::aws:policy/AdministratorAccess`
 
+You may need to include the `--trust` option when bootstrapping the production account as shown below. This command is run from the AWS account that has been configured in the property `accountProd` in *packages/threat-composer -infra/cdk.context.json*.
```
Comment (jessieweiyi) [inline README.md:270]: The bootstrapping is also applied to dev account. 

Suggestion:
```
You need to bootstrap the dev account and production account if they are different from the pipeline AWS account. This command is run using the credentials of the AWS account that has been configured in the property `accountDev` or `accountProd` in *packages/threat-composer -infra/cdk.context.json*.

File: README.md [hunk 2]
```diff
@@ -267,6 +267,10 @@ This deployment option does create resources beyond what is created in the ‘St
...
+
+`cdk bootstrap aws://<prod-aws-account-id>/us-west-2 --cloudformation-execution-policies arn:aws:iam::aws:policy/AdministratorAccess --trust <pipeline-aws-account-id>`
```
Comment (jessieweiyi) [inline README.md:272]: You may need to include the `--trust` option when bootstrapping the production account as shown below. This command is run from the AWS account that has been configured in the property `accountProd` in *packages/threat-composer -infra/cdk.context.json*.

Comment (worldofdub) [reply README.md]: That seems better/more accurate. Have updated README to reflect

**Review by jessieweiyi — APPROVED** [review]

**Comment by jessieweiyi** [comment]
Hi @worldofdub  thanks for your contribution to Threat Composer.

**Pull request merged**